### PR TITLE
Fix typo in system module

### PR DESCRIPTION
**Pull request by kmiya**
Fix typo in system module.

@ito-san 
The followings are just my humble suggestion. Do you have any other good words to replace them?
- entried -> appended
- spinned -> spin()

### CODE REVIEW

File: system/system_monitor/src/net_monitor/net_monitor.cpp [hunk 1]
```diff
@@ -124,7 +124,7 @@ void NetMonitor::checkUsage(diagnostic_updater::DiagnosticStatusWrapper & stat)
 
       ++index;
       close(fd);
-      entried.push_back(ifa->ifa_name);
+      appended.push_back(ifa->ifa_name);
```
Comment (kenji-miyake) [inline system/system_monitor/src/net_monitor/net_monitor.cpp:127]: I think `interface_names` is better.

File: system/system_monitor/include/system_monitor/utils.hpp [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 /// This function will spin the given node and update the diagnostic state at each iteration
 /// \tparam MonitorT Monitor type
-/// \param monitor_ptr Shared pointer of a monitor node to be spinned.
+/// \param monitor_ptr Shared pointer of a monitor node to be spin().
 /// \param period Spin period.
```
Comment (kenji-miyake) [inline system/system_monitor/include/system_monitor/utils.hpp:23]: How about moving this right before the function (around Line32)?

Comment (kenji-miyake) [reply system/system_monitor/include/system_monitor/utils.hpp:23]: Also, using `spin() in already spinning node` is not allowed in ROS2, so we should confirm whether we can still use this function or not.

**Review by ito-san — APPROVED** [review]
LGTM other than comments made by @kenji-miyake.

File: system/system_monitor/src/net_monitor/net_monitor.cpp (hunk 1 above)
Comment (kmiya) [reply system/system_monitor/src/net_monitor/net_monitor.cpp]: Changed in f06568c1dc43c07e8b54747d0c09b02bf0a088c5

File: system/system_monitor/include/system_monitor/utils.hpp (hunk 2 above)
Comment (kmiya) [reply system/system_monitor/include/system_monitor/utils.hpp]: Moved comments in 3213d564af40f917e23acc3ab6108b66c4afe77d (and slightly formatted).
Deleting the function might be out-of-scope of this PR. Better to do on another PR?

Comment (kenji-miyake) [reply system/system_monitor/include/system_monitor/utils.hpp]: @kmiya Thank you!
> Deleting the function might be out-of-scope of this PR. Better to do on another PR?

Yes, it's fine.

**Comment by kenji-miyake** [comment]
@kmiya Could you confirm the failure reason of CI and fix it?

**Comment by kmiya** [comment]
Tests have passed :tada:

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**